python/lsst/ts/standardscripts/maintel/laser_tracker/measure.py

```diff
@@ -0,0 +1,172 @@
+from lsst.ts.observatory.control import RemoteGroup
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+from lsst.ts.observatory.control.remote_group import Usages
+
+
+# TODO (DM-38880) - Subsitute by class in idl.enum when available
+class AlignComponent(enum.IntEnum):
+    M2 = 1
+    Camera = 3
+
+
+class Measure(salobj.BaseScript):
+    """Measure component using laser tracker.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+    - "Starting alignment procedure.": Starting alignment with laser tracker.
+    - "M2 Hexapod aligned with laser tracker.": M2 aligned.
+    - "Camera Hexapod aligned with laser tracker.": Camera measured.
+    """
+
+    def __init__(self, index: int, add_remotes: bool = True):
+        super().__init__(index, descr="Measure MTCS components with laser tracker.")
+
+        self.laser_tracker = RemoteGroup(
+            domain=self.domain,
+            components=["LaserTracker:1"],
+            intended_usage=None if add_remotes else Usages.DryTest,
+            log=self.log,
+        )
+
+        self.mtcs = MTCS(
+            domain=self.domain,
+            intended_usage=None if add_remotes else MTCSUsages.DryTest,
+            log=self.log,
+        )
+
+        self.timeout_measure = 120
+        self.timeout_std = 60
+        self.timeout_short = 5
+
+        self.target = None
+
+    @classmethod
+    def get_schema(cls):
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/laser_tracker/measure.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: MaintelLaserTrackerAlign v1
+        description: Configuration for Maintel laser tracker measurement SAL Script.
+        type: object
+        properties:
+            target:
+                description: >-
+                    Target to measure. Options are:
+                    M2: Secondary mirror.
+                    Camera: LSST Camera.
+                type: string
+                enum: ["Camera", "M2"]
+        additionalProperties: false
+        required:
+            - target
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        self.target = getattr(AlignComponent, config.target)
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = self.timeout_measure + self.timeout_std
+
+    async def measure_target(self):
+        """Measure target with laser tracker."""
+
+        self.laser_tracker.rem.lasertracker_1.evt_offsetsPublish.flush()
+
+        await self.laser_tracker.rem.lasertracker_1.cmd_align.set_start(
+            target=self.target,
+            timeout=self.timeout_measure,
+        )
+
+        try:
```

Comment:
Do you really need to do this? The CSC is already publishing the information I don't think you need to log it from the Script.